pep-9999.rst

```diff
@@ -35,21 +35,22 @@ stubs serve multiple purposes:
   members.
 
 This PEP aims to give guidance to both authors of type stubs and developers
-of type checkers. It describes the constructs that can be used safely in type stubs,
+of type checkers and other tools. It describes the constructs that can be used safely in type stubs,
```

Comment:
"Type checkers" are mentioned throughout the PEP, not just in these three places. I'd suggest either adding "and other tools" only here and leaving it as just "type checkers" everywhere else, or replacing "type checkers" with "tools" everywhere, rather than changing some appearances and not others.

Reply:
I think it's easier to mention it just once at the beginning. I also left in one mention of "tool authors".